src/components/button/button.stories.mdx

```diff
@@ -105,9 +111,48 @@ The `content_start`/`content_end` blocks override the
   <Story name="Custom Icon">{(args) => iconButtonCustomDemo(args)}</Story>
 </Canvas>
 
+## Toggling button
+
+Pass an `aria_pressed` value (`'true'` or `'false'`) to enable a toggling button. If no `content_start_icon`/`content_end_icon` or `content_start`/`content_end` block content is passed in, the `'notifications'` icon will be used by default as the `content_start_icon` value.
+
+You can override the `content_start_icon` value to a different icon.
+
+<Canvas>
+  <Story
+    name="Toggling Button"
+    args={{
+      aria_pressed: 'true',
+    }}
+  >
+    {(args) => button(args)}
+  </Story>
+  <Story
+    name="Secondary Toggling Button"
+    args={{
+      class: 'c-button--secondary',
+      aria_pressed: 'true',
+    }}
+  >
+    {(args) => button(args)}
+  </Story>
+  <Story
+    name="Tertiary Toggling Button"
+    args={{
+      class: 'c-button--tertiary',
+      aria_pressed: 'true',
+    }}
+  >
+    {(args) => button(args)}
+  </Story>
+  <Story name="Custom Icon Toggling Button" args={{ aria_pressed: 'true' }}>
+    {(args) => iconButtonCustomDemo(args)}
+  </Story>
+</Canvas>
+
 ## Template Properties
 
-- `aria_expanded` (boolean): Sets the button's `aria-expanded` attribute.
+- `aria_expanded` (string, `'true'`/`'false'`): Sets the button's `aria-expanded` attribute.
+- `aria_pressed` (string, `'true'`/`'false'`): Sets the button's `aria-pressed` attribute.
```

Comment:
Thoughts on these being `string` values? I'm now questioning whether these should be `boolean` values. I'd need to add extra logic in the template, but it might make more sense.

Reply:
Twig supports [ternary operators](https://twig.symfony.com/doc/3.x/templates.html#other-operators) if you did want to do this:

```twig
<button aria-pressed="{{aria_pressed ? 'true' : 'false'}}">
  ...
</button>
```

Reply:
> Thoughts on these being string values? I'm now questioning whether these should be boolean values. I'd need to add extra logic in the template to convert the value from a boolean to a string, but it might make more sense.

Good question... I'm fine with these being strings, since I think they're treated as strings by assistive technologies (e.g. I think `aria-pressed` !== `aria-pressed="true"`)

If we did switch them to booleans they'd need to be nullish booleans so that you can have the three different states: `null`, `true`, `false`. I don't know how Twig works well enough to know if there's an easy way to do that in Twig.

Reply:
> If we did switch them to booleans they'd need to be nullish booleans so that you can have the three different states: `null`, `true`, `false`. I don't know how Twig works well enough to know if there's an easy way to do that in Twig.

Yeah, I figured I'd start with strings so that I didn't have to get too caught up with nullish boolean values within the context of Twig. I could explore it a bit more, but I'm still unsure if it's worth the extra time or not. Part of me feels it's better to match the native HTML experience (where the `aria-pressed` attribute expects string values of `'true'` or `'false'`).

Reply:
For now, I'm going to leave this as string values since there wasn't a strong argument otherwise. 😉